src/backend/XmlHttpRequest.ts

```diff
@@ -31,6 +33,10 @@ export interface XmlHttpRequestOptions {
   // Used as the URL prefix for fetched files.
   // Default: Fetch files relative to `url`.
   baseUrl?: string;
+  // Whether to use Fetch or XmlHttpRequest.
+  // Can be 'xhr', 'fetch', or 'auto'.
+  // Default: 'auto' which uses fetch if its available, and XHR as a fallback
```

Comment:
Eh... I kind of foresee that scenario. What if you want to use XHR, unless it's not available? I mean, that's basically where we are with Service Workers right? Both  "forceXHR" and a "forceFetch" could end up screwing us over if that feature is unavailable but the other one is. What about "preferXHR"? (where preferXHR = true means prefer XHR, fallback to fetch and preferXHR = false means prefer fetch, fallback to XHR)

But one reason I didn't make it a boolean was I foresee something after fetch. Maybe something with streams. I feel like using a boolean to define how to handle what is really 4 different cases (neither available, just xhr, just fetch, or both) is putting us at the limit of what a single boolean config can do and any future stuff will require adding a *new* option instead of more string variations.